### PR TITLE
Added contacts list UI

### LINKED ISSUE #5: Create list contacts UI

### PR DESCRIPTION
**Pull request by devukurup**
Fixes #5.

image

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/Table/index.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          bordered={false}
+          columnData={columnData}
+          currentPageNumber={currentPageNumber}
+          defaultPageSize={DEFAULT_PAGE_SIZE}
+          rowData={rowData}
+          selectedRowKeys={selectedRowIds}
+          totalCount={rowData.length}
+          handlePageChange={nextPageNumber =>
+            setCurrentPageNumber(nextPageNumber)
+          }
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Table/index.jsx:35]:
```suggestion
 handlePageChange={setCurrentPageNumber}
```

File: app/javascript/src/components/Dashboard/Contacts/Table/index.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          rowClassName={(_, index) =>
+            classnames({ "neeto-ui-bg-gray-100": isOdd(index) })
+          }
+          onRowSelect={ids => setSelectedRowIds(ids)}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Table/index.jsx:39]: Same as above.

**Review by yedhink — COMMENTED** [review]
@devukurup

**Comment by devukurup** [comment]
The above suggestions are resolved in #17.

**Pull request merged**